spec/system/opinion_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'End of module feedback form' do
```

Comment:
I want us to leverage the testing abstraction if possible. That might mean YAML by hand for simplicity